src/Sql/dbo/Tables/UserSigningKey.sql

```diff
@@ -0,0 +1,11 @@
+﻿CREATE TABLE [dbo].[UserSigningKeys] (
```

Comment:
```suggestion
﻿CREATE TABLE [dbo].[UserSigningKey] (
```

Saw this too -- all singular.